src/pages/Teams/Teams.svelte

```diff
@@ -175,7 +175,8 @@
       workspaceObj._id = response.data.data._id;
       workspaceObj.id = response.data.data._id;
       workspaceObj.name = response.data.data.name;
-      workspaceObj.description = response.data.data?.description ?? workspaceObj.description;
```

Comment:
fetch out this response.data.data to another constant and then use it everywhere